src/Traits/SoftekGetterTrait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+namespace DexproSolutionsGmbh\SoftekBarcodeWrapper\Traits;
+
+use FFI\CData;
+
+/**
+ * This Trait can be implemented in each business logic class like transaction classes, service classes
+ * or controller classes. Be sure that the global properties instance and ffi are declared and usable.
+ * Be sure that {@link SoftekSetterTrait::setLicenseKey()} is called before any getter is used.
+ * Notice: Most times getter functions within the barcode tool kit requires a premature function call of mtScanBarCode
+ */
+trait SoftekGetterTrait
```

Comment:
Are these traits ment for public use? Or are they internal to this package? If the latter, please mark these classes with `@internal`.